anyway/widgets/all_locations_widgets/accident_count_by_severity_widget.py

```diff
@@ -61,33 +61,32 @@ def get_accident_count_by_severity(location_info, start_time, end_time):
     def localize_items(request_params: RequestParams, items: Dict) -> Dict:
         if request_params.resolution == BE_CONST.ResolutionCategories.SUBURBAN_ROAD:
             items["data"]["text"] = {
-                "title": _("Number of accidents by severity")
-                + f" - {request_params.location_info['road_segment_name']}"
+                "title": _("Number of accidents by severity"),
+                "subtitle": _(request_params.location_info['road_segment_name'])
```

Comment:
Hi @adirbru , I apologize for providing wrong answer. The names of road segments, streets, and yishuv are not translated.
So as the subtitle holds only the location, it should not be translated. If it would hold additional text, that text would be translated.

Reply:
@ziv17 I think it might be ok?
If no translation is found, original text will be kept right?
And once we'll add translations it should work?

Reply:
Yes, you are right. I think the most important is to test it on both English and Hebrew.

Reply:
@ziv17 I did send the two requests(both with lang=heb and lang=en) and the only changes were in the titles (hope you can see it here, minimized it so it could fit).
<img width="1280" alt="image" src="https://user-images.githubusercontent.com/64323632/214602453-4a103eda-aede-43af-8b11-252af6c83a26.png">

So once I fix the conflict this PR can be approved?